substrate/frame/nomination-pools/src/tests.rs

```diff
@@ -459,6 +464,61 @@ mod reward_pool {
 		});
 	}
 
+	#[test]
+	fn pool_owner_can_adjust_deposit() {
+		ExtBuilder::default().max_members_per_pool(Some(5)).build_and_execute(|| {
```

Comment:
Nit / style suggestion:
I personally find tests easier to read when they follow a GIVEN / WHEN / THEN structure.

Not required at all, but if you ever want to adopt it, here’s a nice example from the [nomination pools tests](https://github.com/paritytech/polkadot-sdk/blob/55e20d5f386efd2e6e102e2ccfd425cbe5aad2d8/substrate/frame/nomination-pools/src/tests.rs#L7636).

It also names the test accounts (alice, bob, etc.) which helps readability.

Totally optional though.